src/AcceptanceTests/Receiving/Issue_532_When_incoming_message_lock_token_is_lost_in_send_atomic_with_receive_mode.cs

```diff
@@ -0,0 +1,92 @@
+namespace NServiceBus.Azure.Transports.WindowsAzureServiceBus.AcceptanceTests.Routing
+{
+    using System;
+    using System.Threading.Tasks;
+    using AcceptanceTesting;
+    using NServiceBus.AcceptanceTests;
+    using NServiceBus.AcceptanceTests.EndpointTemplates;
+    using NUnit.Framework;
+    using AcceptanceTesting.Customization;
+
+    public class Issue_532_When_incoming_message_lock_token_is_lost_in_send_atomic_with_receive_mode : NServiceBusAcceptanceTest
```

Comment:
Ha-ha! Know I know what's @danielmarbach`s OCD thing is.
Pushing that button...now 😛